### PR TITLE
Feature/66 profile screens component implementation

### LINKED ISSUE #66: Profile Screens component implementation
# REQUIREMENTS
This issue consists in the profile screen component implementations.

# DESIGN
image
image

# Acceptance Criteria
List all required acceptance criteria
- [ ] The screen must have it is corresponding component.
- [ ] The provided screen must look like the provided design.

### PR DESCRIPTION
**Pull request by Ing-Heber**
# Description
I created the profile and edit profile screens and added the delete account or bussiness modal component.
 
 Examples:
 - What?: I've added support for authentication to implement Key Result 2 of OKR1. It includes 
model, table, controller and test. For more background, see ticket #JIRA-123.
 - Why?: These changes complete the user login and account creation experience. See #JIRA-123 for more information.
 
 # Testing
1. Download my branch feature/66-profile-screens-component-implementation
2. git pull
3. npm install
4. import them on app 
image

image

 
image

  
  ## Screenshots 
image
image

image
image

### CODE REVIEW

**Review by tovl1 — APPROVED** [review]

**Review by hearagmu — APPROVED** [review]
Excellent!!!

File: tribomobile/components/CheckBoxCustom.js [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
+import Colors from '../src/Colors';
 
-function CheckboxCustom ({title,value}) {
+function CheckboxCustom({customStyle, onChange, title, value}) {
   return (
-    <View style={styles.checkBoxView}>
-      <CheckBox checked={value} color={Colors.BlueLight}/>
-      <Text style={{marginLeft: 15}}>{title}</Text>
+    <View style={[customStyle, styles.checkBoxView]}>
+      {/* <CheckBox checked={value} color={Colors.BlueLight}/>
```
Comment (hearagmu) [inline tribomobile/components/CheckBoxCustom.js:9]: Delete the empty object prior to merge, unless it is usable later.

**Review by hearagmu — APPROVED** [review]
Good customization!

**Review by JordiGN — APPROVED** [review]
Nice man, I like how you used the logo and the line beneath it

**Review by develiasdaniel — APPROVED** [review]
Good design heber

File: tribomobile/components/CheckBoxCustom.js (hunk 1 above)
Comment (Ing-Heber) [reply tribomobile/components/CheckBoxCustom.js:9]: Thank you for noticing it! I will probably use it, nice job

**Pull request merged**